README.md

```diff
@@ -26,6 +26,34 @@ task = Task.new(work_block: proc {
 queue.add_task_async(task: task)
 ```
 
+## Recreatable Tasks
```

Comment:
For some reason when I press `view` this part isn't getting formatted properly. Maybe the headers need to be terminated? `## Recreatable Tasks ##`

Reply:
weird.... 😮